nxt/cli.py

```diff
@@ -110,19 +110,29 @@ def execute(args):
         start = args.start
 
     param_arg_count = len(parameter_list)
-    if param_arg_count % 2 != 0:
-        raise Exception('Invalid parameters supplied, must be in pattern '
-                        '-/node.attr value')
-    parameters = {}
-    i = 0
-    for _ in range(int(param_arg_count / 2)):
-        key = parameter_list[i]
-        if not key.startswith('/'):
-            raise Exception('Invalid attr path key {}, must be '
-                            'formatted as /node.attr'.format(key))
-        val = parameter_list[i + 1]
-        parameters[key] = val
-        i += 2
+    if param_arg_count == 1:
+        # Read file for parameters
+        param_path = parameter_list[0]
+        if not os.path.isfile(param_path):
+            msg = "Parameters file {} does not exist".format(param_path)
```

Comment:
I like the specificity in the other exception message where it tells you the two expected shapes. 
Consider adding more verbiage here too.
```suggestion
            msg = 'Single parameter passed, expected it to be valid parameters file. However, "{}" does not exist'.format(param_path)
```